### PR TITLE
Add Russian language

### PR DESCRIPTION
**Pull request by KagamineP**

### CODE REVIEW

File: src/main/kotlin/ui/strings/Strings.kt [hunk 1]
```diff
@@ -364,23 +432,27 @@ enum class Strings(val en: String, val ja: String, val zhCN: String) {
...
-        zhCN = "忽略参数，使导入更快"
+        zhCN = "忽略参数，使导入更快",
+        ru = "Игнорировать подробные параметры для ускорения импорта"
     );
 
     fun get(language: Language): String = when (language) {
         English -> en
         Japanese -> ja
-        SimplifiedChinese -> zhCN
+        SimplifiedChinese -> zhCN,
```
Comment (sdercolin) [inline src/main/kotlin/ui/strings/Strings.kt:454]: You don't need this comma.

**Comment by sdercolin** [comment]
If you think using English feedback form as a fallback is acceptable, you can just pass the `en` value.
If you want to make a Russian form, I will share the google form with you.
(Please join the [discord server](https://discord.gg/TyEcQ6P73y) for details.)

**Comment by KagamineP** [comment]
> If you want to make a Russian form, I will share the google form with you.
 
Yes I want to make a Russian form

> (Please join the [discord server](https://discord.gg/TyEcQ6P73y) for details.)

Joined

**Comment by sdercolin** [comment]
LGTM, thanks!

**Review by sdercolin — APPROVED** [review]

**Pull request merged**